src/lib/components/Profile.svelte

```diff
@@ -0,0 +1,402 @@
+<script lang="ts">
+	import type { ResponseProfile } from '../../utils/get.type';
+	import type { ActionData } from '../../../.svelte-kit/types/src/routes/profil/$types';
```

Comment:
The import path should probably just be `./$types`.

Reply:
No because it's a components, and it can't access the generated routes path. That's also a reason of why I wanted to rework the modal system.

Reply:
Oh right. Then you should probably define manually the type here, to avoid importing `$types`.